Ginger/GingerCoreNET/Reports/ReportHelper/RepoCoreItem.cs

```diff
@@ -122,8 +120,8 @@ public bool Send_Outlook(bool actualSend = true, string MailTo = null, string Ev
         }
 
         public void ShowAutoRunWindow()
-        {
-            throw new NotImplementedException();
+        {            
+            Reporter.ToConsole(eLogLevel.INFO, "Show UI is set to true but not supported when running with GingerConsole");
```

Comment:
Need to be Reporter.ToLog (to be documented) + eLogLevel should be "Warn"/"Error"

Reply:
Fixed - ToLog+warning